### PR TITLE
Added Cloud SQL user example

### PR DESCRIPTION
**Pull request by betsy-lichtenberg**
Intending to add here:

https://cloud.google.com/sql/docs/sqlserver/create-manage-users
https://cloud.google.com/sql/docs/postgres/create-manage-users
https://cloud.google.com/sql/docs/mysql/create-manage-users

```release-note:none
```

### CODE REVIEW

File: mmv1/products/cgc/terraform.yaml [hunk 1]
```diff
@@ -56,6 +56,7 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         vars:
           database_instance_name: "sqlserver-instance"
           deletion_protection: "true"
+          user: "user"
```
Comment (megan07) [inline mmv1/products/cgc/terraform.yaml:59]: We won't be able to use `random_password` for our VCR tests, so you minding these lines here for this example please?
```
        # Random provider
        skip_vcr: true
```

**Review by megan07 — APPROVED** [review]
Just an issue with the VCR test, if you wouldn't mind adding the note and `skip_vcr` I'll approve and merge. Thanks so much!

File: mmv1/products/cgc/terraform.yaml (hunk 1 above)
Comment (betsy-lichtenberg) [reply mmv1/products/cgc/terraform.yaml:61]: Done

**Pull request merged**